app/views/layouts/application.html.erb

```diff
@@ -56,13 +56,16 @@
       orcid_url: '<%= raw(ORCID_URL) %>',
       ror_url: '<%= raw(ROR_URL) %>',
       datacite: <%= raw(
-        ['ContributorType', 'RelationType', 'RelatedIdentifierType'].map { |name|
+        ['RelationType', 'RelatedIdentifierType'].map { |name|
```

Comment:
We should also handle RelationType and RelatedIdentifierType in a similar way to ContributorType, but we'll do that via separate issue and PR (see https://github.com/pulibrary/pdc_describe/issues/1013)